### PR TITLE
Improve styles for user avatar alt texts

### LINKED ISSUE #171: Improve styling of profile avatar alt texts
I love the alt text feature, but what regularly bothers me is the slightly distorted (oval) display of user avatars:

User avatar distorted through display of alt text

I think it would look a lot nicer to style avatar alt text something like this:

Non-distorted user avatar with alt text below the picture

Thoughts on this?
I'd be happy to attach a PR.

### PR DESCRIPTION
**Pull request by loilo**
Resolves #171

### CODE REVIEW

File: source/style/content.css [hunk 1]
```diff
@@ -221,6 +221,27 @@ code.refined-twitter_markdown {
 	max-height: initial !important;
 }
 
+/* Alternative text for user avatars */
+.twitter_image-alt_profile-container {
```
Comment (filipekiss) [inline source/style/content.css]: You add `refined-twitter_image-alt_profile-container` but these don't have the `refined-` prefix :)
```suggestion
.refined-twitter_image-alt_profile-container {
```

File: source/style/content.css [hunk 2]
```diff
@@ -221,6 +221,27 @@ code.refined-twitter_markdown {
...
+	border-radius: 0 !important;
+	background: transparent !important;
+	overflow: visible !important;
+}
+.twitter_image-alt_profile-container .refined-twitter_image-alt_img {
```
Comment (filipekiss) [inline source/style/content.css]:
```suggestion
.refined-twitter_image-alt_profile-container .refined-twitter_image-alt_img {
```

File: source/style/content.css [hunk 3]
```diff
@@ -221,6 +221,27 @@ code.refined-twitter_markdown {
...
+	border-radius: 50%;
+}
+.twitter_image-alt_profile-container .refined-twitter_image-alt {
```
Comment (filipekiss) [inline source/style/content.css]:
```suggestion
.refined-twitter_image-alt_profile-container .refined-twitter_image-alt {
```

**Review by filipekiss — CHANGES_REQUESTED** [review]

**Comment by filipekiss** [comment]
Nice one, @Loilo!

I just think you missed the prefix on the CSS side of things, so I suggested the changes!

As soon as you finish this, I'll test the extension!

Cheers!

File: source/style/content.css (hunk 1 above)
Comment (loilo) [reply source/style/content.css]: Aw snap. You're absolutely right, I fixed that locally in the `/distribution` instead of the `/source`. 🙈

**Comment by loilo** [comment]
Okay, b992200 should do it. 😁

**Comment by filipekiss** [comment]
Seems fine to me! Great job, @Loilo!

Google Chrome 2019-03-11 at 11 58 57

**Review by filipekiss — APPROVED** [review]

**Comment by loilo** [comment]
My pleasure. 🙂

**Pull request merged**